marathon_acme/clients/vault.py

```diff
@@ -39,6 +50,23 @@ def __init__(self, url, token, *args, **kwargs):
         super(VaultClient, self).__init__(*args, url=url, **kwargs)
         self._token = token
 
+    @classmethod
+    def from_environ(cls, reactor=None, env=os.environ):
+        # Support a limited number of the available config options:
+        # https://github.com/hashicorp/vault/blob/v0.11.2/api/client.go#L28-L40
+        address = _get_environ_str(env, 'VAULT_ADDR', 'https://127.0.0.1:8200')
+        # This seems to be what the Vault CLI defaults to
+        token = _get_environ_str(env, 'VAULT_TOKEN', 'TEST')
+
+        insecure = _get_environ_bool(env, 'VAULT_SKIP_VERIFY')
+        ca_cert = _get_environ_str(env, 'VAULT_CACERT')
+        client_cert = _get_environ_str(env, 'VAULT_CLIENT_CERT')
```

Comment:
Although using custom client-side certs at this point in time is not critical, we do need the option to specify a CA certificate.